pkg/backend/diy/postgres/README.md

```diff
@@ -0,0 +1,76 @@
+# PostgreSQL Backend for Pulumi
+
+This package provides a PostgreSQL-based backend implementation for Pulumi state storage. It stores Pulumi state in a PostgreSQL database table, allowing you to use PostgreSQL as your state storage mechanism.
+
+## Features
+
+- Store Pulumi state in a PostgreSQL database
+- Use standard PostgreSQL authentication and connection options
+- Supports all Pulumi stack operations
+- Configurable table name
+
+## Connection String Format
+
+The PostgreSQL backend connection string follows the standard PostgreSQL connection string format with a `postgres://` prefix:
+
+```
+postgres://username:password@hostname:port/database?param1=value1&param2=value2
+```
+
+## Configuration Options
+
+The following query parameters are supported in the connection string:
+
+- `table`: The name of the table to use for state storage (default: `pulumi_state`)
+- All standard PostgreSQL connection parameters (sslmode, connect_timeout, etc.)
+
+## Usage
+
+To use PostgreSQL as your Pulumi state backend, use the `--backend` flag with the `pulumi` command:
+
+```bash
+pulumi login --backend postgres://username:password@hostname:port/database?sslmode=disable
+
+# Or with a specific table name
+pulumi login --backend postgres://username:password@hostname:port/database?sslmode=disable&table=mystatestable
```

Comment:
`pulumi login` doesn't support a `--backend` flag.

Also we should document all the possible flags, or link to them.  And preferable not suggest `sslmode=disable` I think.